cluster/gce/config-default.sh

```diff
@@ -83,7 +83,7 @@ fi
 # you are updating the os image versions, update this variable.
 # Also please update corresponding image for node e2e at:
```

Comment:
Also update https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml?

Reply:
i am not able to access this file.